### PR TITLE
Feature/week-three-fetch

### PR DESCRIPTION
**Pull request by austinteshuba**
**Uses content from Feature/week-three-servlet - review that PR first.** 

Stubs a response from Java Servlet ("Hello Austin!") and displays it on the webpage (Photo Attached - see below Let's Find Out button.)
Screenshot 2020-06-01 at 4 22 17 PM

Completes step two of week-three walkthrough.

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -32,3 +32,15 @@ function addActivity() {
...
   activityContainer.innerText = activity;
 }
+
+/*
+  This function will execute GET request on the /data URL and display results in 
+  response-container div element 
+*/
+function getData() {
+    fetch("/data")
+        .then(response => response.text())
```
Comment (gbuenoandrade) [inline portfolio/src/main/webapp/script.js:42]: This `then` syntax is really neat!

**Review by gbuenoandrade — APPROVED** [review]
LGTM

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -20,13 +20,15 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/*
```
Comment (antmar) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:23]: Keep this comment formatted as a Javadoc. It's required for public classes.
https://google.github.io/styleguide/javaguide.html#s7.3-javadoc-where-required

Comment (austinteshuba) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:23]: Fixed!

Comment (antmar) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:23]: You need the `/**` at the start, to make this a javadoc.

**Pull request merged**